packagers/yarn2-without-pnp/script.ts

```diff
@@ -0,0 +1,28 @@
+import { PrismaClient } from '@prisma/client'
+
+const client = new PrismaClient()
+
+async function main() {
+  await client.user.deleteMany({})
+
+  const rand = Math.random().toString()
```

Comment:
Why is this needed when you are using `deleteMany`? We don't do this in any of the other tests yet if your aim is idempotent tests https://github.com/prisma/e2e-tests/issues/991

Reply:
Hey @divyenduz, This test is just a modified version of the yarn test, which uses the exact same `script.ts`. I will change it if you think it's nessacary